esti/main_test.go

```diff
@@ -58,239 +88,187 @@ func (i *arrayFlags) Set(value string) error {
 	return nil
 }
 
-const (
-	DefaultAdminAccessKeyId     = "AKIAIOSFDNN7EXAMPLEQ"
-	DefaultAdminSecretAccessKey = "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
-)
-
-var (
-	logger      logging.Logger
-	client      api.ClientWithResponsesInterface
-	endpointURL string
-	svc         *s3.S3
-	server      *webhookServer
-
-	testDirectDataAccess = Booleans{false}
+func envCleanup(client api.ClientWithResponsesInterface, repositoriesToKeep, groupsToKeep, usersToKeep, policiesToKeep arrayFlags) error {
+	ctx := context.Background()
+	errRepos := deleteAllRepositories(ctx, client, repositoriesToKeep)
+	errGroups := deleteAllGroups(ctx, client, groupsToKeep)
+	errPolicies := deleteAllPolicies(ctx, client, policiesToKeep)
+	errUsers := deleteAllUsers(ctx, client, usersToKeep)
+	return multierror.Append(errRepos, errGroups, errPolicies, errUsers).ErrorOrNil()
+}
 
-	metaclientJarPath  string
-	sparkImageTag      string
-	repositoriesToKeep arrayFlags
-	groupsToKeep       arrayFlags
-	usersToKeep        arrayFlags
-	policiesToKeep     arrayFlags
+func deleteAllRepositories(ctx context.Context, client api.ClientWithResponsesInterface, repositoriesToKeep arrayFlags) error {
```

Comment:
It will be more efficient to use a `map` for `repositoriesToKeep` (and for `groupsToKeep`, `usersToKeep` and `policiesToKeep` as well).

Reply:
Don't think it is worth it - lookup for a small size array vs maintain code and convert to map for faster lookup where it is not required.
Note that I prefer to remove all this code if possible - but I need to understand if it is still in use.